content/paths/folder_lock__post_folder_locks.yml

```diff
@@ -0,0 +1,104 @@
+---
+operationId: post_folder_locks
+
+summary: Create folder lock on folder
+
+tags:
+  - Folder Locks
+
+x-box-tag: folder_locks
+x-box-sanitized: true
+x-box-enable-explorer: true
+
+description: |-
+  Creates a folder lock.
+
+requestBody:
+  content:
+    application/json:
+      schema:
+        type: object
+        required:
+          - folder
+        properties:
+          locked_operations:
+            type: object
+            description: |-
+              The operations to lock on the folder. If `locked_operations` is
+              included in the request, both `move` and `delete` must also be
+              included and both set to `true`.
```

Comment:
This makes it sound like I can only set them both to true. I thought I could choose either to be set to true, is this not right?

Reply:
Unfortunately not. The way this is being released as a first iteration both move and delete need to be set to true. This was constructed to be able to expand functionality to additional operations in the future, but at the moment that locked_operations object is fully options and doesn't change the function of the endpoint (i.e. both move and delete need to be set operations to true)

Reply:
👍🏻